src/main/java/com/team/buddyya/certification/dto/response/CertificationResponse.java

```diff
@@ -1,6 +1,6 @@
 package com.team.buddyya.certification.dto.response;
 
-public record EmailCertificationResponse(
+public record CertificationResponse(
         boolean success
 ) {
 }
```

Comment:
p2: 저희 dto에서 값이 하나면 그냥
 ```suggestion
public record CertificationResponse(boolean success) {
}
```
이렇게 한줄로 쓴다고 정했던 것 같아요!